bootstrap.sh

```diff
@@ -1 +1,30 @@
 #!/bin/bash
+
+echo "Creating namespace..."
+kubectl apply -f ./.infrastructure/namespace.yml
+
+echo "Applying ConfigMap..."
+kubectl apply -f ./.infrastructure/configMap.yml
```

Comment:
Ensure that the file path for the ConfigMap is correct. The script currently references `configMap.yml`, which should match the actual file name in your infrastructure directory. Verify that there is no typo in the file name.